docs/src/global/breakpointReducer.js

```diff
@@ -27,7 +27,7 @@ const breakpoint = (state = DEFAULT_STATE, action) => {
                 isXSmall: action.result.isXSmall,
                 isXXSmall: action.result.isXXSmall,
                 winWidth: action.result.winWidth,
-            });
+            };
```

Comment:
Nice!  That's a handy thing ESLINT can auto-fix.  I've been doing it manually.  Getting good at it, but I'm likely no match for ESLINT auto fix! 😁